### DISCUSSION TITLE
Questions about freshwater distribution and mixed layer depth

### DISCUSSION THREAD

**Discussion by mankoff** [NASA-GISS/freshwater-forcing-workshop#23] [Model implementation]
@twilamoon-science and I just had a chat and raised some questions for the group.

We're thinking of recommending freshwater distribution following a vertical profile, not uniform over the top X m. Something based on Fig. 4 from https://doi.org/10.1029/2018GL077000.

![image](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/145117/835fc76e-6e6b-43f4-a9d4-28cc77e9d630)

Some questions this raises are:

+ [ ] We think the distribution should be limited to the mixed layer (ML), above the ML depth (MLD). Is that something that can be specified in the protocol, or is it too ambiguous and we should recommend depth in units m? Or do we need to recommend pressure level if that's the coordinate system of the model?

Can any ocean modeller comment on the following: How quickly would this spread throughout the ML? If we recommend spreading water from 30 to 100 m under the assumption that the MLD is always > 100, would this spread throughout the ML reasonably quickly? Our concern is that if we recommend spreading water from 30 to 200, but the MLD is at 100 or 150, we're now putting significant volumes of freshwater below the MLD which may be less ideal than putting above the MLD hoping to spread throughout the ML.

+ [ ] Is a 2D profile recommendation problematic compared to "spread evenly over top X m" or "spread evenly over meters X to Y"?

+ [ ] Is it problematic to suggest a surface cap w/o freshwater as seen in the `SMW` line above? Hence the "X to Y" m in the previous question.

+ [ ] How do people feel about freshwater and heat having different spatial (z) recommendations?

In Greenland, melting of icebergs (heat transfer) and freshwater from icebergs (mass flow) probably occur in roughly the same place, at least as far as a model is concerned.

In Antarctica there are large tabular icebergs with significant surface area at depth which is also where the warmest waters are found. This makes me think that heat should be extracted from the ocean at depth, even if meltwater is injected near the surface above the MLD.

Does this sound reasonable and tractable to modelers, or are the observationalists getting lost in details and adding unnecessary complexity?

**Comment by trackow** [comment]
In my view, we could recommend (1) vertical profiles & maps (3D approach) and leave it to the modelling teams how to include them. A second and more conservative tier could be (2) to just use the vertically integrated (2D) freshwater distribution (that we could also provide to make sure they are fully consistent/conserving total flux compared to approach 1 and modelling teams don't need to deal with it). For example, when simply dumping everything into the first layers of an ocean model my expectation is that it will mix quickly throughout the mixed layer anyway (might depend somewhat on the model), and the 2D distribution would be enough for that approach. 

> putting significant volumes of freshwater below the MLD which may be less ideal than putting above the MLD hoping to spread throughout the ML

Can you elaborate what would be not ideal? I think for approach (1) it could be a strength and not a weakness that the 3D distribution can reach below the ocean models' mixed layer - as giant tabular icebergs with drafts around 250m+ can do exactly that. This is scientifically most interesting as it could have effects in models that are difficult to foresee (enhanced mixing, changed outcropping, ...) and could therefore be most promising. A coupling of three-dimensional melt fields to the ocean hydrography could strongly affect the vertical structure of the upper water column. In terms of implementation, "spreading evenly over top X m" or "spreading evenly over meters X to Y" is probably easier to implement than a profile as model levels won't match, but is of course something that can be done as well. If modelling teams experience issues with the 3D approach, they could resort to approach 2 and dump everything in the upper levels.

> In Antarctica there are large tabular icebergs with significant surface area at depth which is also where the warmest waters are found. This makes me think that heat should be extracted from the ocean at depth, even if meltwater is injected near the surface above the MLD.

That's a very interesting point! For giant icebergs this is certainly a good idea, but as they calve on decadal timescales and as we are combining small, medium, and giant iceberg freshwater into a single recommended field this might need a bit more discussion probably.

**Reply by mankoff** [reply]
The 'not ideal to put large volumes below the MLD' comment was for Greenland, where we assume most iceberg meltwater ends up above the MLD. And it's a question - maybe we want to spread it more throughout the entire column. But Greenlandic fjord obs suggests it goes to the surface.

In Antarctica, the bergs reach below the MLD for sure, and that is where we should draw heat from, but I have no idea where the meltwater goes. My intuition (often wrong) suggests it rises along the iceberg edge and enters the water column much higher than its melt depth, but may still be below the MLD. Or not, as MLD is deeper in Antarctica (I think).

Writing of heat, Gladstone (2001) reports that Jenkins (1999) `estimates a verticalheat flux local to icebergs in the Southern Ocean of between 150 300 W m^{-2}`

**Comment by nicojourdain** [comment]
Some ocean models uniformly distribute the freshwater over X meters, where X is specified (possibly depending on location, i.e. prescribed as a map). I guess that putting all the freshwater at the surface can result in a thin layer of freshwater in (rare?) cases where winds or sea ice production are not strong enough to overcome the freshwater stratification.

I agree with Thomas that we could put the freshwater below the mixed layer if that's what icebergs do. 

Do we have such melting profiles for Antarctica? It may be possible to save it in new simulations, but that won't be ready any time soon. @trackow do you have this in your existing simulations? 

About this:
> In Antarctica there are large tabular icebergs with significant surface area at depth which is also where the warmest waters are found. This makes me think that heat should be extracted from the ocean at depth, even if meltwater is injected near the surface above the MLD.

I am not sure it is a good idea to separate the latent heat from the freshwater. First, this will require some modifications in the ocean models. Second, the latent heat cools the water mass at the same time as it freshens, so the buoyant water mass carries the heat lost through melting. Therefore, I think that heat and freshwater should be put at the same depth.

**Reply by mankoff** [reply]
I agree that separating heat and mass sounds dangerous. But we should be clear that the largest heat sink - the phase transition - cools deeper waters. If we choose to distribute the freshwater closer to the surface (which we should), we're cooling the wrong water. That's OK, or it is what it is until icebergs are fully resolved.

**Reply by nicojourdain** [reply]
Plumes transport both heat and freshwater. The phase transition cools and freshens the plume, and both heat and freshwater are transported by the plume to upper layers. There is entrainment between the plume and the ambiant water, for both freshwater and heat. So in my opinion, both freshwater and heat should be injected at the same location.

**Comment by gavinny** [comment]
Absolutely - mass/energy and tracer fluxes need to be physically coherent in each of the specific cases. .

But let’s try and keep the different terms clear. The spatial maps from Nicolás et al are for icebergs melting in the open ocean, and the melt profile (s) we use will not care about the MLD. The fluxes at the fjord mouth (from Greenland mostly) could be set to be uniform over a certain depth or over the mixed layer. While basal fluxes at the ice shelf calving front should be at the depth of the ice shelf.

We should also expect that done groups might chose to simplify things for ease of coding, and so guidance if uniform depth distributions are going  to be used will be helpful.

The main impetus right now is to have something rather than expect it to be perfect.

Gavin


From: "Nicolas C. Jourdain" ***@***.******@***.***>>
Subject: [EXTERNAL] [BULK] Re: [NASA-GISS/freshwater-forcing-workshop] Questions about freshwater distribution and mixed layer depth (Discussion #23)
Date: 16 May 2024 09:21
To: "NASA-GISS/freshwater-forcing-workshop" ***@***.******@***.***>>
Cc: "Subscribed" ***@***.******@***.***>>

CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.




Plumes transport both heat and freshwater. The phase transition cools and freshens the plume, and both heat and freshwater are transported by the plume to upper layers. There is entrainment between the plume and the ambiant water, for both freshwater and heat. So in my opinion, both freshwater and heat should be injected at the same location.

**Comment by twilamoon-science** [comment]
@mankoff - Not sure where best to be helpful next. Seems you are bringing several of the various data sources and ideas together, and might have a sense of (very) specific next needs. I think our recent conversation, discussion here, and [concept outline](https://docs.google.com/presentation/d/1Nhic0I6hx8b7xBy7VXIh2Hqum_8B7OLZR4YbUOkQ2io/edit?usp=sharing) provide a solid foundation. ?

@gavinny - Also if there are specific, updated requests on the paper front. ?

Thanks!

**Reply by mankoff** [reply]
Twila, I've tried to articulate specific needs in this spreadsheet: https://docs.google.com/spreadsheets/d/1wCPz-SogA_cbsM9Moi2OLJP_KuIfLAeOtOX-g6BTx98/edit?usp=sharing - but it's not that specific. Just data that matches what we're writing about in the text.

I've added columns for point-of-contact for each row, tried to split things up to be very granular (e.g., not confounding the source magnitude, say icebergs, with the xy or z distribution, all of which might come from different people). I've added a new row on 'progress' because I've completed some items.
